core/src/test/java/org/ehcache/core/UserManagedCacheTest.java

```diff
@@ -100,7 +94,6 @@ private void assertFailingTransitionGoesToLowestStatus(InternalCache cache) thro
     } catch (StateTransitionException e) {
       assertThat(cache.getStatus(), is(Status.UNINITIALIZED));
     }
-
```

Comment:
Sorry, I meant could we leave the imports in place since nothing else was modified in the class.